src/components/MenuDetails.js

```diff
@@ -0,0 +1,174 @@
+
+  function handleTabs(e) {
+    if (e === "tab1") {
+      settab1("columns-2 text-center bg-white");
+      settab2("columns-2 text-center");
+      settab3("columns-2 text-center");
+      settab1Content(true);
+      settab2Content(false);
+      settab3Content(false);
+    } else if (e === "tab2") {
+      settab1("columns-2 text-center");
+      settab2("columns-2 text-center bg-white");
+      settab3("columns-2 text-center");
+      settab1Content(false);
+      settab2Content(true);
+      settab3Content(false);
+    } else if (e === "tab3") {
+      settab1("columns-2 text-center");
+      settab2("columns-2 text-center");
+      settab3("columns-2 text-center bg-white");
+      settab1Content(false);
+      settab2Content(false);
+      settab3Content(true);
+    }
+  }
+
+  return (
+    <div classname="mr-5">
+      <div
+        style={{ display: "flex", justifyContent: "center" }}
+        className="ml-4"
+      >
+        <div className="mr-20">
+          <div className="flex ">
+            <div className="tab1 cursor-pointer" onClick={() => handleTabs("tab1")}>
+              <p className="res-details-tab-text">
+                <button
+                  type="button"
+                  className="bg-white hover:bg-gray-200 text-black h-12 px-11 mb-0 mr-10"
+                >
+                  <span>Menu </span>
+                  <div className="ml-3">
+                    <MdOutlineRestaurant />
+                  </div>
+                </button>
+              </p>
+            </div>
+            <div className="tab2 cursor-pointer" onClick={() => handleTabs("tab2")}>
+              <p className="res-details-tab-text">
+                <button
+                  type="button"
+                  className="bg-white hover:bg-gray-200 text-black h-12 px-9 mx-24"
+                >
+                  Reviews{" "}
+                  <div className="ml-5">
+                    <MdReviews />
+                  </div>
+                </button>
+              </p>
+            </div>
+            <div className="tab3 cursor-pointer" onClick={() => handleTabs("tab3")}>
+              <p className="res-details-tab-text">
+                <button
+                  type="button"
+                  className="bg-white hover:bg-gray-200 text-black h-12 px-11"
+                >
+                  Info{" "}
+                  <div className="ml-2">
+                    <AiFillInfoCircle />
+                  </div>
+                </button>
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+      {tab1Content && (
+        <div>
+          <div className="m-1 flex flex-col items-center bg-white md:flex-row md:max-w-xl  bg-white-800 ml-9">
+          <div className="ml-3">
+                <BiSearch />
+              </div>
+            <div className="flex items-center mb-2 mt-2">
+              <input
+                type="text"
+                className="form-control search-menu-input"

```

Comment:
This is not in tailwind.css. So please convert it to the tailwind. We need to convert "form-control" to "form-input" because form-control is a bootstrap class.

Reply:
Working on the changes.

Reply:
Fixed.

Reply:
Check now